src/functionalTest/resources/features/F-204 - Get NoC Questions/common/204_NoC_WithoutEvents_Case_Creation_With_Org_Policies.td.json

```diff
@@ -0,0 +1,276 @@
+{
+  "_guid_": "204_NoC_WithoutEvents_Case_Creation_With_Org_Policies",
```

Comment:
You could have extended NoC_WithoutEvents_Case_Creation_With_Org_Policies__Base to simplify this file.